modules/rgbd/src/tsdf.cpp

```diff
@@ -388,27 +436,38 @@ struct IntegrateInvoker : ParallelLoopBody
                     Point2f projected = proj(camSpacePt, camPixVec);
 
                     depthType v = bilinearDepth(depth, projected);
-                    if(v == 0)
+                    if (v == 0) {
                         continue;
-
-                    // norm(camPixVec) produces double which is too slow
-                    float pixNorm = sqrt(camPixVec.dot(camPixVec));
+                    }
+                    float pixNorm;
+                    if (false)
```

Comment:
Please replace this `false` by a meaningful const bool

Reply:
done